app/domain/operations/fdsh/ssa/h3/ssa_verification_response_processor.rb

```diff
@@ -9,19 +9,49 @@ module Ssa
       module H3
         # This class will update the person and consumer role based on response
         class SsaVerificationResponseProcessor
-          send(:include, Dry::Monads[:result, :do, :try])
-          include EventSource::Command
+          include ::Operations::Transmittable::TransmittableUtils
 
           def call(params)
+            transmittable_objects = yield find_transmittable(params)
             person = yield find_person(params[:person_hbx_id])
             consumer_role = yield store_response_and_get_consumer_role(person, params[:response])
             updated_consumer_role = yield update_consumer_role(consumer_role, params[:response])
-
+            _updated_transmittable_objects = yield record_results(transmittable_objects)
             Success(updated_consumer_role)
           end
 
           private
 
+          # we should feature flag X
+          # for all services, check if success or failure X
+          # if failure response, find job (don't record if not found?), save failure X
+          # if success response:
+          # * find job
+          # * create response transmission and transaction, save the response payload
+          # * link to the subject
+          # * process the response using existing operation(s) X
+          # * mark the transmittable objects as succeeded X
+          #
+          # Questions:
+          # generic operation(s) or not? - yes for the successful response
+          # what to do if job not found for a successful response? create job and process as before just recording the response
+          # when to close out the job if more than one response expected, would it have separate operation?
+
+
+          def find_transmittable(params)
+            return Success() unless EnrollRegistry[:ssa_h3].setting(:use_transmittable).item == "true"
+            result = Operations::Transmittable::GenerateResponseObjects.new.call({job_id: params[:metadata]&.job_id,
+                                                                                  key: :ssa_verification_response,
+                                                                                  payload: params[:response],
+                                                                                  correlation_id: params[:person_hbx_id],
```

Comment:
Let's talk about this tomorrow, we might be able to do something with passing gid rather than the these two params